packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -243,13 +245,45 @@ const WorkflowsView = defineComponent({
 		sendFiltersTelemetry(source: string) {
 			(this.$refs.layout as IResourcesListLayoutInstance).sendFiltersTelemetry(source);
 		},
+		saveFiltersOnQueryString() {
+			const queryString = new URLSearchParams();
+
+			if (this.filters.search) {
+				queryString.append('search', this.filters.search);
+			}
+
+			if (this.filters.status !== undefined && this.filters.status !== '') {
+				queryString.append('status', this.filters.status);
+			}
+
+			if (this.filters.tags.length) {
+				queryString.append('tags', this.filters.tags.join(','));
+			}
+
+			void this.$router.push({
+				name: VIEWS.WORKFLOWS,
+				query: { ...Object.fromEntries(queryString) },
+			});
+		},
+		setFiltersFromQueryString() {
+			const { tags, status, search } = this.$route.query;
+
+			this.filters = {
```

Comment:
should we validate values before applying them? what happens if tag id does not exist, for example?